src/main/java/uk/gov/indexer/dao/CurrentKeysUpdateDao.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.indexer.dao;
+
+import org.skife.jdbi.v2.sqlobject.Bind;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+
+interface CurrentKeysUpdateDao {
+    String CURRENT_KEYS_TABLE = "CURRENT_KEYS";
+
+
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS " + CURRENT_KEYS_TABLE + " (ID INTEGER PRIMARY KEY, KEY VARCHAR UNIQUE)")
+    void ensureCurrentKeysTableExists();
+
+
+    @SqlUpdate("INSERT INTO " + CURRENT_KEYS_TABLE + "(ID, KEY) VALUES(:id, :key)")
```

Comment:
can we start using "serial number" rather than "id" to be consistent with the terminology in our glossary?